.github/update.py

```diff
@@ -18,6 +18,19 @@
 LCONVERT = 'lconvert'
 
 # Make sure the Target ts files are up to date
+# Generate a translations.pri containing all languages in this repo
+qtTranslationProject = open(os.path.join(VPN_PROJECT_DIR,"translations","translations.pri"), "w")
```

Comment:
A few nits:
* With Python3, it's more common to use `with open() as f:`, so you don't have to close the file handler.
* Let's stick to one type of quotes (single) and spacing between argument. You can run `flake8` on the script to see what it reports.

See the changes that I suggested in 
https://github.com/mozilla-mobile/mozilla-vpn-client/issues/72

In particular about cutting regions at 2 characters.

Reply:
Thanks for the feedback! :) 
- ran flake8 all good now except "line too long"; for the shell stuff i guess that's okay. 
- single quotes everywhere
- now using `with open() as f:`
- dropped the check if the folder is an iso tag, now using a simple ignorelist.